jbmc/src/java_bytecode/README.md

```diff
@@ -43,6 +43,45 @@ To be documented.
 
 To be documented.
 
+\section java-bytecode-remove-instanceof Remove instanceof
+
+Java has an operator called `instanceof`, which is used to check if an instance of a class is of a particular type. For example given that `A` and `B` are the identifiers of classes, which derive directly from Object. If the following example java code is executed -
```

Comment:
I think a summary of the operator is useful - but I'd probably stop short of the full inheritance example (you could exapnd your initial example to include `a instanceof Object`. You can instead link to this:https://docs.oracle.com/javase/tutorial/java/nutsandbolts/op2.html (irritatingly you can't link to the `instanceof` section)